modules/renter/hostdb/hostweight.go

```diff
@@ -311,6 +312,7 @@ func (hdb *HostDB) uptimeAdjustments(entry modules.HostDBEntry) float64 {
 	downtime := entry.HistoricDowntime
 	uptime := entry.HistoricUptime
 	recentTime := entry.ScanHistory[0].Timestamp
```

Comment:
If we decide to get rid of the `Timestamp` field we can probably also get rid of that line and check the order later using the blockheight instead. We should wait for @DavidVorick to give some input on this though.